packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -36,7 +36,7 @@ module.exports = (resolve, rootDir) => {
     testURL: 'http://localhost',
     transform: {
       '^.+\\.css$': resolve('config/jest/cssTransform.js'),
-      '^.+\\.tsx?$': resolve('config/jest/typescriptTransform.js'),
+      '^.+\\.tsx?$': resolve('node_modules/ts-jest/preprocessor.js'),
```

Comment:
Now using the former file as a `require`-wrapper to work around the problems with different root paths.